tasks/managed/collect-data/collect-data.yaml

```diff
@@ -72,6 +81,12 @@ spec:
     - name: releasePipelineMetadata
       type: string
       description: json object containing git resolver metadata about the running release pipeline
+    - description: Produced trusted data artifact
```

Comment:
It's better to use the same attribute order with the previous params.

Reply:
fixed